src/main/java/org/openhmis/webservice/AuthenticationService.java

```diff
@@ -69,8 +70,10 @@ public String getExternalId(@HeaderParam("Authorization") String authorization,
 		if(!Authentication.googleAuthenticate(authorization, Authentication.READ))
                         throw new AccessDeniedException();
                 log.info("POST /externalId/ " + id_token);
-                String externalId = Authentication.resolveIdentity(id_token);
-                return externalId;
+                // Send an external ID and a user object, if the user
```

Comment:
This comment confused me a little bit.  It says "Send an external ID and a user object"... but we're clearly just returning an AccountDTO.  The documentation for AccountDTO should explain that type; if we redocument it here, then if we change AccountDTO later (say, by adding new fields), we'll accidentally make comments like this obsolete or at least slightly out of date.